swagger.yaml

```diff
@@ -6196,6 +6239,11 @@ definitions:
     type: object
     description: IP address block for node IP blocks.
     properties:
+      cluster_id:
+        type: string
+        format: uuid
+        description: The cluster that this network is associated with.
```

Comment:
If there is a primary key only for cluster-id, then there will be only a single record per cluster in this table - which is not what we want.